RELEASE-NOTES.txt

```diff
@@ -2,6 +2,10 @@ Unreleased
 ------
 * [*] Disabled featured image banner on iOS. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3474] 
 
+1.52.2
+------
+* [*] Disabled featured image banner on iOS. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3474] 
```

Comment:
Looks like this entry is duplicated in the `Unreleased` section, since `Disabled featured image banner on iOS.` change was introduced in `1.52.2`, we could remove the one from `Unreleased`, right?

Reply:
@jd-alexander I've removed the duplicated entry and update the file.